src/app/pages/meditationPlayer/MeditationPlayer.tsx

```diff
@@ -32,9 +33,19 @@ export default function MeditationPlayer({
     isPlaying ? '' : playGong();
   }
 
+  function endMeditation() {
+    const oldSessions = JSON.parse(localStorage.getItem('sessions') || '');
```

Comment:
When there is no item you need to return a default value like so:
```suggestion
    const oldSessions = JSON.parse(localStorage.getItem('sessions')) || [];
```
Also you want to return an array because you can't spread a string

Reply:
I had the exact same situation. If the local storage is empty, both version throw an error. The combination of both seems to work:
`const oldItinerary = JSON.parse(
      window.localStorage.getItem('itinerary') || '[]'
    );`

Reply:
@derjubu @phlgr I will then implement Julians suggestions :)